pom.xml

```diff
@@ -1055,6 +1055,7 @@
                 <module>tensorflow-java</module>
                 <module>xstream</module>
                 <module>webrtc</module>
+				<module>persistence-modules/java-mongodb</module> <!-- long running -->
```

Comment:
there are tabs in this line, please revert back to spaces and remove the comment "long running"